### PR TITLE
Update devDependencies

### PR DESCRIPTION
**Pull request by fernahh**
Update:

- eslint;
- eslint-config-standard;
- karma;
- karma-jasmine.

@ContaAzul/domphysis

### CODE REVIEW

**Comment by cacontinuos** [comment]
@fernahh, thanks for your PR! By analyzing the blame information on this pull request, we identified @matheuspoleza to be a potential reviewer

**Comment by YosephCode** [comment]
++

File: .eslintrc.json [hunk 1]
```diff
@@ -45,7 +48,7 @@
     "one-var": [2, "never"],
     "quotes": [2, "single"],
     "semi": [2, "always"],
-    "space-after-keywords": 2,
+    "keyword-spacing": 2,
```
Comment (matheuspoleza) [inline .eslintrc.json]: why this change?

File: .eslintrc.json [hunk 2]
```diff
@@ -2,6 +2,9 @@
   "ecmaFeatures": {
     "modules": true
   },
+  "parserOptions": {
+    "sourceType": "module"
+  },
```
Comment (matheuspoleza) [inline .eslintrc.json]: and this?

**Comment by fernahh** [comment]
@matheuspoleza This changes are because the update of `eslint-config-standard` and `eslint` were major type.

**Comment by cirocfc** [comment]
++

**Comment by matheuspoleza** [comment]
+1 @fernahh
